docs/DeveloperGuide.adoc

```diff
@@ -838,18 +838,52 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |register a new volunteer |begin tracking their volunteer work & hours
```

Comment:
Should user be 'Administrator of a Social Welfare Organization' instead?

Reply:
Decided to leave it as user and perhaps define the intended user to be the administrator elsewhere instead - since 'admin' might be a little misleading. What do you guys think?